enhanced-resolve/lib/Storage.d.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="node" />
+import { CommonFileSystemMethod } from './common-types';
+
+declare class Storage {
+    duration: number;
+    running: Object;
```

Comment:
`Object` should usually be avoided. `any` and `{}` both describe all values, but `any` tells the compiler not to type-check uses (so it can be used `any`-where), where as you can't do much with `{}` without downcasting first. More info [here](http://stackoverflow.com/questions/18961203/typescript-any-vs-object).